scripts/all-accounts-aws-config.ini

```diff
@@ -0,0 +1,33 @@
+; NOTE this is meant to have ALL accounts we use along with canonical account names
+;      (as used by scripts in this folder)
+; NO SCRIPT SHOULD AUTOMATICALLY RUN ACROSS ALL THESE AS ANY ACCOUNTS MAY BE ADDED HERE
+
+;
+; sessions define the two sets of credentials needed for our accounts
+;
+
+[sso-session umccr-bootstrap-session]
+sso_start_url = https://umccr.awsapps.com/start
+sso_region = ap-southeast-2
+sso_registration_scopes = sso:account:access
+
+[sso-session unimelb-bootstrap-session]
+sso_start_url = https://unimelb.awsapps.com/start
+sso_region = ap-southeast-2
+sso_registration_scopes = sso:account:access
+
+;
+; profiles define every account in our organisations
+;
+
+[profile umccr-dev]
+sso_session = umccr-bootstrap-session
+sso_account_id = 843407916570
+sso_role_name = AdministratorAccess
+region = ap-southeast-2
+
+[profile unimelb-australiangenomics]
+sso_session = unimelb-bootstrap-session
+sso_account_id = 258177526432
+sso_role_name = AWSAdministratorAccess
+region = ap-southeast-2
```

Comment:
So if we are going to do this kind of script (also see the security scripts) - we probably should have some common master definition of all of this

Reply:
Yep. Agree, thought of the same.

Reply:
A master JSON that we can JQ into various forms??

Reply:
Sure